report/data_sources/hubspot/client.py

```diff
@@ -338,7 +338,7 @@ def _get_objects(cls, accessor, fields: Iterable[Field]):
     def _map_item(cls, item, fields: Iterable[Field]):
         result = {}
         for field in fields:
-            value = item.properties[field.hs_field] or None
+            value = item.properties.get(field.hs_field) or None
```

Comment:
Not equivalent to just the `get`. The `or` here coerces falsy values (0, None, '') all to None. The tests expect this behaviour.